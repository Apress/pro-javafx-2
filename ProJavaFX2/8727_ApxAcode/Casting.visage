/*
 * @test
 * @run
 */

class A {
  function f() {
    println("A.f() called.");
  }
}

class B extends A {
  function g() {
    println("B.g() called.");
  }
}

var a: A;
a = B {};
a.f();
(a as B).g();

