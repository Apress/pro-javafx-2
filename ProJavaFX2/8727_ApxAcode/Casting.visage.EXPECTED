A.f() called.
B.g() called.
